.editorconfig

```diff
@@ -0,0 +1,17 @@
+# top-most EditorConfig file
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+end_of_line = lf
+insert_final_newline = true
+
+
+[*.{css,coffee}]
+indent_size = 2
+trim_trailing_whitespace = true
+
+[*.{js,json}]
+indent_size = 4
```

Comment:
This isn't true for `package.json`

Reply:
Yes, i4 4 spaces > https://github.com/e14n/ofirehose/blob/master/package.json#L5

Reply:
Huh, weird. LGTM though.